src/manifest.js

```diff
@@ -132,6 +132,7 @@ export const forEachMediaGroup = (master, callback) => {
  */
 export const setupMediaPlaylist = ({ playlist, uri, id }) => {
   playlist.id = id;
+  playlist.retryCount = 0;
```

Comment:
perhaps it would be better to call this `retries` to match `maxPlaylistRetries`?

Reply:
Updated here and throughout